docs/semgrep-ci/configuration-reference.md

```diff
@@ -69,17 +69,25 @@ To find more details about some of these configuration options, see the followin
 
 ## Diff-aware scanning (`SEMGREP_BASELINE_REF`)
 
-For [diff-aware scans](overview.md#features), this option filters scan results to those introduced after the git commit, in a branch, or tag. For example, you have a repository with 10 commits. You set the commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10.
+Semgrep scans can be classified by scope. The scope of a scan refers to what lines of code are scanned in a codebase. When classifying scans by scope, there are two types of scans:
+
+<dl>
+    <dt>Full scan</dt>
+    <dd>A full scan will scan your entire codebase and report every finding in the codebase. It is recommended to perform a full scan of your <code>main</code> branch at a regular cadence, such as every night or every week, depending on your own needs. This ensures that Semgrep App has a full list of all findings in your code base, regardless of when they were introduced. To run a full scan, simply run <code>semgrep ci</code> without setting <code>SEMGREP_BASELINE_REF</code> environment variable.</dd>
+    <dt>Diff-aware scan</dt>
+    <dd>A diff-aware scan runs on your code before and after some "baseline" and only reports findings that are newly introduced in the commits after that baseline. For example, imagine you have a repository with 10 commits and you set commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10. This is how <code>semgrep ci</code> should be run in pull requests and merge requests, since it will report only the findings that are created by those code changes. To run a diff-aware scan, use <code>SEMGREP_BASELINE_REF=<span class="placeholder">REF</span> semgrep ci</code> where <span class="placeholder">REF</span> can be a commit hash, branch name, or other git ref.</dd>
```

Comment:
- @chmccreery Ad "should" - `This is how <code>semgrep ci</code> should be run in pull requests and merge requests` - do we want to say: 
A. This is recommended setup for Semgrep in CI.
B. This is how Semgrep should work in CI, but it often does not (uncertainty implied by modal words such as "should").
Avoid these words and rather specify. See https://developers.google.com/style/word-list?hl=en#should.
- @s-santillan I am not sure if we can use `semgrep ci` - some time ago we discussed this with @pabloest that we should rather state Semgrep in CI. I am not sure if there was any additional discussion made that I was not part of regarding this topic.
- git refs are probably [git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References).


```suggestion
    <dd><p>A diff-aware scan runs on your code before and after some "baseline" and only reports findings that are newly introduced in the commits after that baseline.</p>
    <p>For example, imagine a hypothetical repository with 10 commits. You set commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10. This is how Semgrep in CI can run in pull requests and merge requests, since it reports only the findings that are created by those code changes. To run a diff-aware scan, use <code>SEMGREP_BASELINE_REF=<span class="placeholder">REF</span> semgrep ci</code> where <span class="placeholder">REF</span> can be a commit hash, branch name, or other git reference.</p></dd>
```

Reply:
Another note - try to build this, if the output is satisfactory.

Reply:
@adamkvitek Semgrep CI in this context is literally a command, `semgrep ci`.

Reply:
I'm accepting the suggestion but reverting "Semgrep in CI" to `semgrep ci`.